common/app/Map/Map.jsx

```diff
@@ -32,15 +90,27 @@ export class ShowMap extends PureComponent {
 
   render() {
     const { superBlocks } = this.props;
+    const style = {
+      position: 'absolute',
+      top: 0,
+      bottom: 0,
+      left: 0,
+      right: 0,
+      overflowY: 'auto',
+      overflowX: 'hidden'
+    };
```

Comment:
This style block doesn't change. Why does this need to be a prop instead of css?

Reply:
Agreed, will move into .less